simulator/src/ux.js

```diff
@@ -69,9 +69,55 @@ function showContextMenu() {
     $('#contextMenu').css({
         visibility: 'visible',
         opacity: 1,
-        top: `${ctxPos.y}px`,
-        left: `${ctxPos.x}px`,
     });
+    var windowHeight = $(window).height() / 2;
+    var windowWidth = $(window).width() / 2;
+    //for top, left, right, bottom position calculation
+    var t;
```

Comment:
Avoid single letter variables https://www.codereadability.com/i-n-k-single-letter-variable-names/

Reply:
Yes, sure I will change them. Thanks for the reference site.